dashboard/app/controllers/home_controller.rb

```diff
@@ -35,7 +35,11 @@ def health_check
   GALLERY_PER_PAGE = 5
 
   def index
-    redirect_to '/courses'
+    if current_user.try(:admin?)
+      redirect_to '/admin'
```

Comment:
On what pages is the admin extra links available? Are those links available from `/admin`?